DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -271,7 +271,9 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="19">
+  <class name="pat::PackedCandidate" ClassVersion="23">
+    <version ClassVersion="23" checksum="663492232"/>
+    <version ClassVersion="21" checksum="1075665714"/>
```

Comment:
Why is "22" missing here? Later changes could cause problems if someone adds 22 and 23.